docs/anime.md

```diff
@@ -652,3 +652,9 @@ Sources
 ### 最新汉化
 
 <Route author="junfengP" example="/zdfx" path="/zdfx"/>
+
+## アニメ新番組
+
+### 當季新番
+
+<Route author="devinmugen" example="/bangumionline/updates" path="/bangumionline/updates"/>
```

Comment:
I would suggest using the namespace `bangumi` as the domain name is `bangumi`.online but not `bangumionline`.com/.org, etc.

Reply:
`bangumi` usually means `bangumi.tv`, so I have added `online` to distinguish it.

Reply:
You can use something like `/bangumi/online/updates`. In fact, `/bangumi` is used for `bangumi.moe` currently, and routes related to `bangumi.tv` haves their own category like `/bangumi/calendar/today` or `/bangumi/subject`

Reply:
Under V2 does `bangumionline` need to be merged under `bangumi` in order to use `bangumi/online`? Or is there another way?

Reply:
Under V2 does `bangumionline` need to be merged under `bangumi` in order to use `bangumi/online`? Or is there another way?

Reply:
Yes, it needs to be merged in order to use `/bangumi/online`

Reply:
I have changed the route to `bangumi/online`. 
The github-actions bot shows a failure, but it has been successfully deployed on Vercel.
https://rssh-ub-devinmugen.vercel.app/bangumi/online